webapp-django/crashstats/tokens/static/tokens/js/home.js

```diff
@@ -1,14 +1,29 @@
 $(function() {
-    $('p.code button').click(function(event) {
+
+    'use strict';
+
+    $('.token p.code[data-key]').each(function() {
+        var p = $(this);
+        p.prepend(
+            $('<code>')
+            .addClass('truncated')
+            .on('click')
```

Comment:
ha! good catch! I used to have it so that you can click the `<code>` to have the same affect as the `<button>` next to it. But it didn't work because it made it weird when you fumbled and tried to highlight it.